src/LiveComponent/doc/index.rst

```diff
@@ -2741,7 +2741,7 @@ name with ``#[LiveArg]`` in front::
     public function incrementProductCount(#[LiveArg] int $product)
     {
         $this->productCount++;
-        $this->lastProduct = $data['product'];
+        $this->lastProduct = $product;
```

Comment:
```suggestion
        $this->lastProductId = $product;
```

I think this would make the next paragraph easier to understand... what do you think ?